src/views/v2/rsvp/details/availability.php

```diff
@@ -17,18 +17,44 @@
  * @version TBD
  */
 
-$is_unlimited = -1 === $rsvp->remaining();
+use Tribe__Date_Utils as Dates;
 
+$is_unlimited = - 1 === $rsvp->remaining();
+$is_in_stock  = $rsvp->is_in_stock();
+
+$days_to_rsvp = Dates::date_diff( current_time( 'mysql' ), $rsvp->end_date );
+$days_to_rsvp = floor( $days_to_rsvp );
+
+// Only show Days to RSVP if it is happening within the next week and is in stock.
+if ( ! $is_in_stock || 6 < $days_to_rsvp ) {
+	$days_to_rsvp = false;
+}
 ?>
 <div class="tribe-tickets__rsvp-availability tribe-common-h6 tribe-common-h--alt tribe-common-b3--min-medium">
-	<?php if ( ! $rsvp->is_in_stock() ) : ?>
+	<?php if ( ! $is_in_stock ) : ?>
 		<?php $this->template( 'v2/rsvp/details/availability/full', [ 'rsvp' => $rsvp ] ); ?>
 	<?php elseif ( $is_unlimited ) : ?>
-		<?php $this->template( 'v2/rsvp/details/availability/unlimited', [ 'is_unlimited' => $is_unlimited ] ); ?>
+		<?php
+		$this->template( 'v2/rsvp/details/availability/unlimited', [
+			'is_unlimited' => $is_unlimited,
+			'days_to_rsvp' => $days_to_rsvp,
+		] );
+		?>
 	<?php else : ?>
-		<?php $this->template( 'v2/rsvp/details/availability/remaining', [ 'rsvp' => $rsvp ] ); ?>
+		<?php
+		$this->template( 'v2/rsvp/details/availability/remaining', [
+			'rsvp'         => $rsvp,
+			'days_to_rsvp' => $days_to_rsvp,
+		] );
+		?>
 	<?php endif; ?>
 
-	<?php $this->template( 'v2/rsvp/details/availability/days-to-rsvp', [ 'rsvp' => $rsvp ] ); ?>
-
+	<?php if ( false !== $days_to_rsvp ) : ?>
+		<?php
+		$this->template( 'v2/rsvp/details/availability/days-to-rsvp', [
+			'rsvp'         => $rsvp,
+			'days_to_rsvp' => $days_to_rsvp,
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 6d977844368edbd0000e2a6404e3ab90</sub>